.github/workflows/build-test.yml

```diff
@@ -1,5 +1,5 @@
 name: build-test
-on: [push, pull_request]
+on: [push]
```

Comment:
```suggestion
on: [push, pull_request]
```

Reply:
I will add that back when the PR is in a ready state.